src/qualimap/qualimap_rnaseq/config.vsh.yaml

```
@@ -0,0 +1,117 @@
+name: "qualimap"
+keywords: [RNA-seq, quality control, QC Report]
+description: |
+  Qualimap RNA-seq QC reports quality control metrics and bias estimations 
+  which are specific for whole transcriptome sequencing, including reads genomic 
+  origin, junction analysis, transcript coverage and 5’-3’ bias computation.
+links:
+  homepage: http://qualimap.conesalab.org/
+  documentation: http://qualimap.conesalab.org/doc_html/analysis.html#rna-seq-qc
+  issue_tracker: https://bitbucket.org/kokonech/qualimap/issues?status=new&status=open
+  repository: https://bitbucket.org/kokonech/qualimap/commits/branch/master
+references:
+  doi: https://10.1093/bioinformatics/btv566
+license: GNU GPL v2
+authors:
+  - __merge__: /src/_authors/dorien_roosen.yaml
+    roles: [ author, maintainer ]
+argument_groups:
+  - name: "Input"
+    arguments: 
+    - name: "--bam"
+      type: file
+      required: true
+      example: alignment.bam
+      description: Path to the sequence alignment file in BAM format, produced by a splicing-aware aligner.
+    - name: "--gtf"
+      type: file
+      required: true
+      example: annotations.gtf
+      description: Path to genomic annotations in Ensembl GTF format.
+
+  - name: "Output"
+    arguments: 
+    - name: "--output"
+      direction: output
+      type: file
+      required: true
+      example: rnaseq_qc_results.txt
+      description: Text file containing the RNAseq QC results.
+    - name: "--output_counts"
+      type: file
+      required: false
+      direction: output
+      description: Output file for computed counts.
+    - name: "--output_report"
+      type: file
+      direction: output
+      required: false
+      example: report.html
+      description: Report output file. Supported formats are PDF or HTML.
```

What would you think about:

```suggestion
    - name: "--qc_results"
      direction: output
      type: file
      required: true
      example: rnaseq_qc_results.txt
      description: Text file containing the RNAseq QC results.
    - name: "--counts"
      type: file
      required: false
      direction: output
      description: Output file for computed counts.
    - name: "--report"
      type: file
      direction: output
      required: false
      example: report.html
      description: Report output file. Supported formats are PDF or HTML.
```